### ISSUE TITLE
Cannot delete seedlots, possibly because they have been used in transactions

### ISSUE THREAD

**Issue by keocorak** [solgenomics/sgn#5012]
Expected Behavior 
--------------------------------------------------------------------------

Cannot delete seedlots, possibly because they have been used in transactions. But dialog is unclear- if I begin deleting seedlots using a list, it asks if I want to delete green highlighted seedlots, if I confirm, 0 seedlots are deleted. 

For Bugs:
---------

### Environment 
https://sugarcane-industry.breedbase.org/

#### Steps to Reproduce
Seedlots/'Delete Seedlots Using A List'/ 
<img width="750" alt="Screenshot 2024-07-15 at 10 31 07 AM" src="https://github.com/user-attachments/assets/6002f15a-2639-41c8-a446-f0444b514d0d">

**Comment by lukasmueller** [comment]
Yes, that's correct, they have been used in transactions and can't be deleted for that reason. Which seedlots need to be deleted? I can do that from the backend.

**Comment by titima15** [comment]
There is another option. You can mark those seedlots as "Discarded". This function will freeze all transaction activities, instead of completely remove from database.

**Comment by keocorak** [comment]
Thank you! In this case I do want to hard remove them but this is a special case where we want to copy some but not all of the data from an existing database to a fresh instance. In typical use cases I think the restrictions on deletion and the ability to 'discard' are important and reasonable. I'll work with Lukas to delete the desired seedlots from the backend.

**Comment by keocorak** [comment]
I will leave this issue open, though because I think the dialog box should inform the user that seedlots were not deleted explicitly because they'd been used in transactions. Right now it just says they weren't deleted without explaining why.
